controllers/release/release_suite_test.go

```diff
@@ -115,3 +118,26 @@ var _ = AfterSuite(func() {
 	err := testEnv.Stop()
 	Expect(err).NotTo(HaveOccurred())
 })
+
+// GetModuleCRDPath return the filesystem path after GOPATH/pkg/mod
+// for pseudo versioned modules.
+func GetModuleCRDPath(goModPath string, moduleName string) string {
+	var CRDPath string
+	readFile, err := os.Open(goModPath)
+	if err != nil {
+		panic(err)
+	}
+	scanner := bufio.NewScanner(readFile)
+	for scanner.Scan() {
+		re := regexp.MustCompile(".*" + moduleName + ".*")
```

Comment:
Wouldn't be this a simple `strings.Contains`?

Reply:
done